src/intent_pilot/utils/models/model_handler.py

```diff
@@ -58,7 +59,7 @@ def call_gpt_4_vision_preview_labeled(
     openai_client,
     messages,
     screenshots_dir: Path = Path("screenshots"),
-    skip_som_draw_labels=[],
+    skip_some_draw_labels=[],
```

Comment:
```suggestion
    skip_som_draw_labels=[],
```
because som = set-of-mark